### ISSUE TITLE
Loading or Creading Singleplayer worlds

### ISSUE THREAD

**Issue by nimraes** [sp614x/optifine#4276]
The game crushes if i load/create worlds on singleplayer.

I use some mods, optifabric, carpet, extracarpet, phosphor, lithium, some datapacks, but these are used on multiplayer server that i own.

**Comment by LOuroboros** [comment]
Same here. Trying to use Optifine via Optifabric makes my single player worlds crash. They load up correctly once I remove them though.
I'm personally using the latest preview release of Optifine *(OptiFine 1.16.1 HD U G2 pre2)* with Fabric Loader v0.8.8 Build 202, Fabric API v0.14.0 Build 371 and Optifabric v1.1.0 Beta 2.

Here's my crash log, in case it helps: https://pastebin.com/nattB41q

One of my worlds is a Superflat Creative one, and at first I thought it got corrupted somehow since I created it back during the snapshot 20w12a, but even though I recreated it on the 1.16.1 it still crashed.
